src/Tomlyn/Model/Accessors/StandardObjectDynamicAccessor.cs

```diff
@@ -165,6 +165,13 @@ public override bool TryGetPropertyType(SourceSpan span, string name, out Type?
             return true;
         }
 
+        // If configured to ignore missing properties on the target type,
+        // return false to indicate it is missing but don't set an error
+        if (Context.IgnoreMissingProperties)

```

Comment:
Could you group this with the Diagnostics.Error code below (so !Context.IgnoreMissingProperties) instead?